src/index.css

```diff
@@ -11,3 +11,133 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
```

Comment:
I like that you spit your CSS code up in different sections using comments, nice!